modules/admin/proc-configuring-postgresql-instance-using-operator.adoc

```diff
@@ -0,0 +1,109 @@
+[id="proc-configuring-postgresql-instance-using-operator_{context}"]
+= Configuring an external PostgreSQL instance using the Operator
+
+You can configure an external PostgreSQL instance by using the {product} Operator. By default, the Operator creates and manages a local instance of PostgreSQL in the same namespace where you have deployed the {product-very-short} instance. However, you can change this default setting to configure an external PostgreSQL database server, for example, Amazon Web Services (AWS) Relational Database Service (RDS) or Azure database.
+
+.Prerequisites
+
+* You are using a supported version of PostgreSQL. For more information, see the link:https://access.redhat.com/support/policy/updates/developerhub[Product life cycle page].
+* You have the following details:
+** `db-host`: Denotes your PostgreSQL instance Domain Name System (DNS) or IP address 
+** `db-port`: Denotes your PostgreSQL instance port number, usually `5432`
+** `username`: Denotes the user name to connect to your PostgreSQL instance
+** `password`: Denotes the password to connect to your PostgreSQL instance
+* You have installed the {product} Operator.
+* Optional: You have a CA certificate, TLS private key, and TLS certificate so that you can secure your database connection by using the Transport Layer Security (TLS) protocol. For more information, refer to your PostgreSQL vendor documentation.
+
+[NOTE]
+====
+By default, {product-short} uses a database for each plugin and automatically creates it if none is found. As an administrator, know that users might need the `Create Database` privilege in addition to `PSQL Database` privileges for configuring an external PostgreSQL instance.  
+====
+ 
+
+.Procedure
+
+. Optional: Create a certificate secret to configure your PostgreSQL instance with a TLS connection:
```

Comment:
I'd suggest adding the full form of TLS in the first instance.

Reply:
expanded at the first instance in the module.